imports/plugins/core/orders/client/templates/workflow/shippingTracking.html

```diff
@@ -13,26 +13,47 @@
       <button class="btn btn-link" data-event-action="editTracking">{{shipment.tracking}}</button>
      {{/if}}
 
-    {{#if isCompleted}}
-      {{i18n "orderShipping.itemsHaveBeenShipped" "Items have been shipped"}}
-      <button class="btn btn-default" data-event-action="resendNotification" data-i18n="orderShipping.resendNotification">Resend Shipment Notification</button>
-    {{else}}
-
-    <hr>
-      {{#if shipmentReady}}
-        <p data-i18n="orderShipping.shippingNotifyCustomer">Notify the customer that their items will be shipping soon.</p>
-        <button class="btn btn-success btn-block" data-event-action="shipmentShipped" data-i18n="orderShipping.shipped">Shipped</button>
+    {{#unless isCompleted}}
+      {{#if isShipped}}
+        {{i18n "orderShipping.itemsHaveBeenShipped" "Items have been shipped"}}
+        <button class="btn btn-default" data-event-action="resendNotification" data-i18n="orderShipping.resendNotification">Resend Shipment Notification</button>
       {{else}}
-        <button
-          class="btn btn-default btn-block"
-          type="button"
-          data-i18n="orderShipping.shipmentPacked"
-          data-event-action="shipmentPacked">All Items Packed, Ready to Ship</button>
+
+      <hr>
+        {{#if shipmentReady}}
+          <p data-i18n="orderShipping.shippingNotifyCustomer">Notify the customer that their items will be shipping soon.</p>
+          <button class="btn btn-success btn-block" data-event-action="shipmentShipped" data-i18n="orderShipping.shipped">Shipped</button>
+        {{else}}
+          <button
+            class="btn btn-default btn-block"
+            type="button"
+            data-i18n="orderShipping.shipmentPacked"
+            data-event-action="shipmentPacked">All Items Packed, Ready to Ship</button>
+        {{/if}}
+      {{/if}}
+    {{/unless}}
+    <br/>
+    {{#unless isCompleted}}
+      {{#if isShipped}}
+        <div class="complete-panel">
           <div style="padding: 10px 0">
-            {{> React CancelOrderButton}}
+            {{> React CompleteOrderButton}}
           </div>
+        </div>
       {{/if}}
-    {{/if}}
-    <br/>
+      <div class="row cancel-panel">
+        <div class="col-xs-6 col-sm12">
+          <select id="cancelation-reason" name="cancelation-reason" class="form-control">
+            <option value="failed-delivery" selected>Failed delivery</option>
+            <option value="wrong-item">Wrong Item</option>
+            <option value="damaged">Damaged in transit</option>
+            <option value="out-of-stock">Item out of stock</option>
```

Comment:
add an extra field that caters for other options not listed above. and provide an input field to cater for details about this option.